sycl/doc/extensions/IntelGPU/IntelGPUDeviceInfo.md

```diff
@@ -11,7 +11,40 @@ This proposal details what is required to provide this information as a SYCL ext
 
 The Feature Test Macro will be defined as:
 
-    #define SYCL_EXT_INTEL_DEVICE_INFO 1
+    #define SYCL_EXT_INTEL_DEVICE_INFO 2
+
+
+# Device UUID #
+
+A new device descriptor will be added which will provide the device Universal Unique ID (UUID).
+
+This new device descriptor is only available for devices in the Level Zero platform, and the matching aspect is only true for those devices. The DPC++ default behavior is to expose GPU devices through the Level Zero platform.
```

Comment:
If we wanted this to be Level-Zero only then we'd add this extension to https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md. My understanding is that the intention is to have it supported at least with OpenCL too, so let's not have this description be too Level-Zero centric.

Reply:
ok, will modify.